delta-iceberg/integration_tests/iceberg_converter.py

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (2021) The Delta Lake Project Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import col
+from delta.tables import DeltaTable
+import shutil
+import random
+
+testRoot = "/tmp/delta-iceberg-converter/"
+warehousePath = testRoot + "iceberg_tables"
+shutil.rmtree(testRoot, ignore_errors=True)
+
+# Enable SQL commands and Update/Delete/Merge for the current spark session.
```

Comment:
update the comment. we are not using any Update/Delete/Merge. Add a comment about iceberg catalog.